Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,23 +1,22 @@
 import React from 'react';
 import './App.css';
 
+//Header Components
+import HeaderContainer from "./components/HeaderComponents/HeaderContainer";
+
+//Card components
+import CardContainer from "./components/CardComponents/CardContainer";
+
+//Footer Components
+
 const App = () => {
   return (
     <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+      <HeaderContainer />
```

Comment:
good job on keeping your app.js clean and simple by importing everything that you needed to import and not adding an extra logic or code when it wasn't needed and instead keeping everything in their relevant components.